scaffolds/src-openid-auth-code-flow/backend/credentials.js

```diff
@@ -0,0 +1,23 @@
+/**
+* client_credentials will store client_id and client_secret credentials per account per cluster
+* Those credentials are used to access FSM oauth API and fetch a FSM access token.
+*/
+const client_credentials = {};
+
+// Load credentials from process.env defined
+const {
+  FSM_CLOUD_HOST, 
+  FSM_ACCOUNT, 
+  FSM_CLIENT_ID, 
+  FSM_CLIENT_SECRET
+} = process.env;
+
+if (FSM_CLOUD_HOST && FSM_ACCOUNT && FSM_CLIENT_ID && FSM_CLIENT_SECRET) {
+  client_credentials[FSM_CLOUD_HOST] = {};
+  client_credentials[FSM_CLOUD_HOST][FSM_ACCOUNT] = {
+    'client_id': FSM_CLIENT_ID,
+    'client_secret': FSM_CLIENT_SECRET
+  };
+}
```

Comment:
Maybe Else throw?
So the container crashes if config is missing - for early feedback and good dx